### ISSUE TITLE
Git Bash Core Dump with Git for Windows 2.4.4.windows2

### ISSUE THREAD

**Issue by mindw** [gitextensions/gitextensions#2849]
How to reproduce:
* Install  GitExtensions 2.48.05
* Install https://github.com/git-for-windows/git/releases/tag/v2.4.4.windows.2
* Open any repository in GitExtensions
* Click on the "Git Bash" icon

Expected:
* a new git bash console window opens

Actual:
A windows pops open briefly and closes immediately. A file named ``bash.exe.stackdump`` is created at the root (see dump below).

Some notes:
* The issue showed up in the 2nd release candidate
* The git install tree was changed (files removed, added etc).

```
Stack trace:
Frame        Function    Args
00000239DF0  00180071E93 (06E006F0069, 00100000030, 7FFA337B0B16, 00000000000)
00000000000  0018007346E (00000000064, 00600000000, 000000002AC, 00000000000)
0000023A4E0  0018012CA4F (0000000000A, 0060003A459, 0000005D064, 0000000001B)
0000023AE40  001800722E4 (00000000000, 00000000000, 0000023B330, 0000023A7F0)
0000023A870  7FFA35A43F0D (00180040000, 00000000000, 00180290480, 0000023A840)
0000023A870  7FFA35A04887 (00000000000, 0060004B8B0, 00000000000, 00000000000)
0000023B618  7FFA35A4309A (0000023B614, 0060004B8B0, 0060003FBE0, 0060004B8B0)
0000023B618  0010049871A (00000000FF0, 00000000FF0, 001004D5850, 0060003FBE0)
0000023B618  001004960FF (00000000001, 00000000000, 001801EF838, 00000000001)
00000000000  0010048FE87 (00000000100, 001004E3246, 001801259DB, 001005E6E68)
001004DE885  001004875D8 (00000000000, 00000000000, 001005E6E68, 001004E3246)
001004DE885  00100471CCD (00000000000, 00000000000, 00000000000, 00000000000)
001004DE885  00100449069 (001005E6E74, 00000000001, 00100425548, 00000000001)
001004DE885  00100402645 (00000000000, 00000000000, 00000000000, 00000000000)
001005E6E68  00100404393 (00000000000, 00000000000, 00000000000, 00000000000)
0000023B990  0010040700A (00000000000, 0060010EFE0, 00000000000, 0000023B990)
End of stack trace (more stack frames may be present) 
```

**Comment by KindDragon** [comment]
I think this is because of issue https://github.com/git-for-windows/git/issues/222

**Comment by mindw** [comment]
Resolved in the fourth release candidate (2.4.5.1).
